IDEAS/BoundaryConditions/Climate/Meteo/Solar/RadSolData.mo

```diff
@@ -78,6 +79,7 @@ equation
       points={{106,-20},{-39.9,-20},{-39.9,30.1}},
       color={0,0,127},
       smooth=Smooth.None));
+  if not (solDataInBus and not outputAngles) then
```

Comment:
this is not strictly necessary since constAngLin is conditionally removed already